demo/image_demo.py

```diff
@@ -55,7 +57,7 @@ def main(args):
 
     is_dir = os.path.isdir(args.img)
```

Comment:
Considering that this function will be used in many places, it is recommended to move it to util and reuse it